### ISSUE TITLE
Mentor User cannot see the Profile 

### ISSUE THREAD

**Issue by robert-dm** [hack4impact-uiuc/mentee#962]
https://github.com/hack4impact-uiuc/mentee/assets/35821871/3bba8167-ba9c-4179-8fff-ebd597c1ccd4

**Comment by robert-dm** [comment]
@Humayun0008 this has been fixed in DEV URL, could you please uncomment this test case?

**Comment by Humayun0008** [comment]
@robert-dm Still not Resolved 

https://github.com/hack4impact-uiuc/mentee/assets/131605337/044d76fb-b484-4cb9-b3e8-d0c6d5ac9117

**Comment by robert-dm** [comment]
The problem was that the user was removed from database, or email updated to testEmail@gmail.com and I have fixed the user in the database, now it is working fine.  Attached video.   

We should not have any other problem from now on, if yes, then there is a process in your local files or anywhere in automated tests affecting the robertmurer+mentor1@gmail.com user in database.

@Humayun0008 

https://github.com/hack4impact-uiuc/mentee/assets/35821871/f6e4cfc2-ab09-411b-aa23-9c44e9022cb6
